packages/input-text/src/TextAreaInput.story.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import { action } from "@storybook/addon-actions";
+import { storiesOf } from "@storybook/react";
+import { withInfo } from "@storybook/addon-info";
+
+import { TextAreaInput } from "./";
+
+storiesOf("Text Area Input", module)
+  .add("Default", withInfo()(() => <TextAreaInput rows="3" />))
+  .add("Focused", withInfo()(() => <TextAreaInput focused rows="3" />))
+  .add(
+    "With label",
+    withInfo()(() => <TextAreaInput label="Field Label" rows="3" />)
+  )
+  .add(
+    "With placeholder",
+    withInfo()(() => (
+      <TextAreaInput
+        label="Field Label"
+        placeholder="input value here"
+        rows="3"
+      />
+    ))
+  )
+  .add(
+    "With value",
+    withInfo()(() => (
+      <TextAreaInput
+        value="a value"
+        label="Field Label"
+        placeholder="input value here"
+        rows="3"
+      />
+    ))
+  )
+
+  .add(
+    "Disabled",
+    withInfo()(() => (
+      <TextAreaInput
+        disabled
+        onChange={action("input changed")}
```

Comment:
Is this line intentional, as the input is disabled?

Reply:
I can remove.